packages/connection/src/class-manager.php

```diff
@@ -26,7 +26,14 @@ class Manager {
 	const SECRETS_EXPIRED        = 'secrets_expired';
 	const SECRETS_OPTION_NAME    = 'jetpack_secrets';
 	const MAGIC_NORMAL_TOKEN_KEY = ';normal;';
-	const JETPACK_MASTER_USER    = true;
+
+	/**
+	 * For internal use only. If you need to get the connection owner, use the provided methods
+	 * get_connection_owner_id, get_connection_owner and is_get_connection_owner
+	 *
+	 * @var boolean
+	 */
+	const MASTER_USER = true;
```

Comment:
See my comment in [this discussion](https://github.com/Automattic/jetpack/pull/16974#discussion_r495133793).

Reply:
Is there a better term than `MASTER_USER`? As we move away (generally) from using `master`, is there something else we can use if we are introducing a new constant.

Reply:
@leogermani replying to https://github.com/Automattic/jetpack/pull/16974/files#r496214560 here

I can appreciate not wanting to have inconsistency, but I think we're introducing future debt there.

As @kbrown9 mentioned, we need to account for `JETPACK_MASTER_USER` for now as a BC issue for consuming projects. If we add `MASTER_USER` and then rename it later, we'll have to account for both of those constants plus the new one.

If we use a new constant now, we can still rename the db option later (or not, part of the point of the package is not having to worry about what the db is storing).

Removing `master` would help move #16099 a little forward.

We could also add a @ todo in the comment to add a `private` visibility declaration once PHP 7.1 is the minimum supported PHP version.